Mathlib/Data/Nat/Prime/Nth.lean

```diff
@@ -0,0 +1,23 @@
+/-
+Copyright (c) 2024 Ralf Stephan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ralf Stephan
+-/
+import Mathlib.Data.Nat.Prime.Defs
+import Mathlib.Data.Nat.Nth
+
+/-!
+# The Nth primes
+-/
+
+namespace Nat
+
+@[simp]
+theorem nth_prime_zero_eq_two : nth Prime 0 = 2 := nth_count prime_two
```

Comment:
This feels like the sort of thing that should be delegated to `norm_num`.

Reply:
I think it is nice like it is.